examples/all_in_one/README.rst

```diff
@@ -0,0 +1,79 @@
+All-in-one example
+==================
+This example is optimized for saving developer time by storing all secrets in a single file (e.g. ``secret.enc.json``).
+
+The downside is any configurations which should be stored side by side might not be.
+
+Getting started
+---------------
+To use this example, run the following
+
+.. code:: bash
+
+    # From the `sops` root directory
+    # Import the test key
+    gpg --import tests/sops_functional_tests_key.asc
+
+    # Navigate to our example directory
+    cd examples/all_in_one
+
+    # Decrypt our secrets
+    bin/decrypt-secrets.sh
+
+    # Optionally edit a secret
+    # bin/edit-secret.sh config/secret.enc.json
+
+    # Run our script
+    python main.py
```

Comment:
It's means to be an example script that relies on the config (e.g. running an application). I will update the comment to make it more clear